notebooks/noj_book/ml_basic.clj

```diff
@@ -191,7 +191,7 @@ split
 
 (loss/classification-accuracy
  (:survived (ds-cat/reverse-map-categorical-xforms (:test split)))
- (:survived (ds-cat/reverse-map-categorical-xforms lreg-prediction)))
+ (:survived  lreg-prediction))
```

Comment:
old code would work as well here.
As no cat-map is attached for tribuo models, the 'reverse-map-categorical-xforms' is a no-op